src/applications/personalization/dashboard/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -148,6 +158,11 @@ const ApplyForBenefits = ({ getESREnrollmentStatus, shouldGetESRStatus }) => {
   );
 };
 
+ApplyForBenefits.propTypes = {
+  getESREnrollmentStatus: PropTypes.func,
+  shouldGetESRStatus: PropTypes.bool,
```

Comment:
NIT: These proptypes could/should be set as .required since you cant really make it work without, at least the `shouldGetESRStatus` boolean